courses/free-courses-hi.md

```diff
@@ -66,6 +67,10 @@
 * [ReactJS Tutorial in Hindi 2020](https://www.youtube.com/playlist?list=PLwGdqUZWnOp3aROg4wypcRhZqJG3ajZWJ) - Vinod Bahadur Thapa (Thapa Technical)
 
 
+### Next.Js
+
+* [NextJS Tutorial in Hindi](https://www.youtube.com/playlist?list=PLwGdqUZWnOp2rDbpfKAeUi9f8qZMS7_cv) - Vinod Bahadur Thapa (Thapa Technical)
+
```

Comment:
There should be an additional line break here.

You can click on the failed build to get additional information on why it's failing. Then you can see exactly what's wrong and how to fix it.